src/PowerShellEditorServices/Debugging/StackFrameDetails.cs

```diff
@@ -58,19 +74,21 @@ public class StackFrameDetails
         /// </param>
         /// <returns>A new instance of the StackFrameDetails class.</returns>
         static internal StackFrameDetails Create(
-            CallStackFrame callStackFrame,
+            PSObject callStackFrameObject,
             VariableContainerDetails autoVariables,
             VariableContainerDetails localVariables)
         {
             return new StackFrameDetails
             {
-                ScriptPath = callStackFrame.ScriptName ?? "<No File>",
-                FunctionName = callStackFrame.FunctionName,
-                LineNumber = callStackFrame.Position.StartLineNumber,
-                ColumnNumber = callStackFrame.Position.StartColumnNumber,
+                ScriptPath = (callStackFrameObject.Properties["ScriptName"].Value as string) ?? NoFileScriptPath,
```

Comment:
I wonder if over time we will want two classes: `LocalStackFrameDetails` and `RemoteStackFrameDetails`.  This class isn't that complicated now so it probably isn't worth it but if it creeps in complexity over time, maybe we reconsider?